app/components/searchworks4/physical_availability_component.html.erb

```diff
@@ -100,9 +99,7 @@
               <div class="d-flex flex-column flex-sm-row gap-2 gap-sm-5 p-2 suppress-item-level-request-button-if-there-is-a-location-level-request-button">
                 <div class="pt-2" data-availability-target="location">
                   <div><%= render LocationComponent.new(location: location, document: document, suppress_off_campus: false, classes: []) %></div>
-                  <% unless document.holdings.libraries.one? && document.holdings.libraries.first.locations.one? %>
```

Comment:
Wasn't this conditional in place to guard against bad records? Why is it being removed?

Reply:
I believe you're thinking about a similar condition that's at the component level (checking if we should go down the single-location path or not). This condition is either gratuitous or wrong (if we're down this branch, we've already established there are multiple locations)